scripts/veracode/static_analysis_init.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+echo "=========================== Starting Static Analysis Init Script ==========================="
+
```

Comment:
Add `set -e` to exit the script when a command fails (returns a non-zero status).

Reply:
The change has been made.